modules/administration-guide/partials/proc_migration-to-devworkspace-engine.adoc

```diff
@@ -0,0 +1,226 @@
+
+[id="migration-to-devworkspace-engine_{context}"]
+= Migration to {devworkspace} engine.
+
+This procedure describes how to migration to {devworkspace} engine using https://docs.openshift.com/container-platform/latest/operators/understanding/olm/olm-understanding-olm.html[OLM] to support the Devfile 2.0.0 file format and mentions how to do so on existing instances.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* An instance of {prod-short} deployed using xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[Operator Hub] from `{prod-channel}` channel on OpenShift cluster version greater or equal to 4.8
+* OpenShift OAuth is enabled. See xref:configuring-openshift-oauth.adoc[].
+* Bundled PostgreSQL
+
+.Procedure
+
+. All workspaces must be stopped and changes pushed back to Git repositories.
+
+. Backup {prod-short} data. See xref:managing-backups-using-chectl.adoc[].
+
+include::partial$snip_scaling-down-che.adoc[]
+
+. Create the script to get all {prod-short} users:
```

Comment:
Rather than displaying a script that the user should edit, could we have a script that can receive parameters and that the user can download without modifications?

Reply:
User should not edit any scrits but copy/paster into console to run.

Reply:
Then the step should be "Download the script from following URL" (URL preferably including the release version)  rather than "Select, copy and paste a script we display in the content of the script in the docs".

Reply:
This script depends on doc templates like {prod-namespace} and {prod-checluster}.
I can't put this script anywhere but doc.

Reply:
Could we have a script where {orch-cli}, {prod-namespace} and {prod-checluster} are arguments? Now we would be embedding untestable code inside the docs, it feels weird.

Reply:
These parameters are something that users should not care about.